src/controller.ts

```diff
@@ -89,6 +87,8 @@ export function acceptExCmd(ex_str: string) {
     // TODO: Errors should go to CommandLine.
     try {
         let [func, args] = exmode_parser(ex_str)
+        if (!ex_str.startsWith("repeat"))
```

Comment:
Better to just test `func == excmds.repeat` or similar. Otherwise you're duplicating the work of the exmode_parser.

Reply:
This is fixed now. Thank you for being so patient!